utils/ofi_debug_ip.patch

```diff
@@ -86,7 +89,15 @@ index 64772fef0..4c0166ae8 100644
  	ep->rx_avail = 0;
  	ofi_bsock_discard(&ep->bsock);
  }
-@@ -561,6 +564,7 @@ static void xnet_ep_cancel_rx(struct xnet_ep *ep, void *context)
+@@ -444,6 +447,7 @@ void xnet_ep_disable(struct xnet_ep *ep, int cm_err, void* err_data,
+ 	if (!xnet_io_uring)
+ 		xnet_halt_sock(xnet_ep2_progress(ep), ep->bsock.sock);
+ 
++	XNET_OUTPUT_ERR_WARN(ep, cm_err);
+ 	ret = ofi_shutdown(ep->bsock.sock, SHUT_RDWR);
```

Comment:
this is more than debug?  Looks like it's calling extra function

Reply:
oh, this is the macro to show the debug message once the ep is disabled, so it will be helpful for debugging purpose, ok, let me try a different name.